lib/lightning_web/live/run_live/streaming.ex

```diff
@@ -210,6 +214,7 @@ defmodule LightningWeb.RunLive.Streaming do
 
       unquote(helpers())
       unquote(handle_infos())
+      @impl true
```

Comment:
One tiny thing here, instead of placing it above the unquote I think this can be inside the handle_asyncs macro above the first `handle_async` - I see thats the pattern with handle_infos...